bin/postgres-ha/pgbackrest/pgbackrest-archive-push-local-s3.sh

```diff
@@ -31,8 +31,17 @@ pgbackrest archive-push $1
 local_exit=$?
 
 # then try S3
-pgbackrest archive-push --repo-type=s3 $1
-s3_exit=$?
+# if TLS verification is disabled, pass in the appropriate flag
+# otherwise, leave the default behavior and verify TLS
+if [[ $PGHA_PGBACKREST_S3_VERIFY_TLS == "false" ]]
+then
+    pgbackrest archive-push --repo1-type=s3 --no-repo1-s3-verify-tls $1
+    s3_exit=$?
+elif
+then
```

Comment:
`else`?

Reply:
Hm looking at this, perhaps we set up something like:

```bash
archive_push_cmd_args=("--repo1-type=s3")

# if TLS verification is disabled, pass in the appropriate flag
# otherwise, leave the default behavior and verify TLS
if [[ $PGHA_PGBACKREST_S3_VERIFY_TLS == "false" ]]
then
    archive_push_cmd_args+=("--no-repo1-s3-verify-tls")
fi

pgbackrest archive-push ${archive_push_cmd_args[*]} $1
s3_exit=$?
```

Reply:
Yes, that should be an else, fixed, thanks.

And the update to the flag handling looks good to me, updated.